src/drivers/camera_trigger/camera_trigger_params.c

```diff
@@ -97,3 +97,14 @@ PARAM_DEFINE_INT32(TRIG_MODE, 0);
  * @group Camera trigger
  */
 PARAM_DEFINE_INT32(TRIG_PINS, 12);
+
+/**
+ * Camera trigger distance
+ *
+ * Sets the distance at which to trigger the camera. TRIG_INTERVAL sets an upper bound on the triggering frequency.
```

Comment:
I agree that it does not sound very intuitive. I tried to catch the case, where the speed and set distance result in a shooting interval that is less than the set activation time. In this case the activation time will be smaller than the one set, which could result in no pictures being taken.